app/src/org/commcare/activities/AdvancedActionsActivity.java

```diff
@@ -223,9 +226,9 @@ private void startConnectionTest() {
         startActivity(i);
     }
 
-    private void clearUserData() {
+    public static void clearUserData(final Activity activity) {
```

Comment:
Yea. Since demo users don't have an options menu, there was no way to clear user data for the demo user once you logged in as them. I ran into wanting to be able to do so numerous times while doing dev, and I imagine it could be useful for others as well.